generators/environment/templates/docker/build.yml

```diff
@@ -145,17 +145,18 @@ services:
     environment:
       APP_DOMAIN: <%= host.local %>
       GDT_DOMAIN: <%= host.local %>
-      # Check https://hub.docker.com/r/phase2/devtools-build/ for other Node version options.
+      # See https://hub.docker.com/r/outrigger/build for other Node version options.
       NODE_VERSION: 6
       <% if (usePLS) { %>
       # Allow for pattern lab compilation need detection with reasonable performance from
       # within the build container.
       # See: https://github.com/paulmillr/chokidar/blob/master/README.md#user-content-performance
-      CHOKIDAR_USEPOLLING: 1
-      <% } -%>
-      # Set to "true" to load xdebug configuration. Note this will cause significant composer
-      # performance degradation.
-      PHP_XDEBUG: "false"
+      CHOKIDAR_USEPOLLING: 1<% } %>
+      # Configure Xdebug
+      # See http://docs.outrigger.sh/common-tasks/using-xdebug-with-phpstorm/
+      PHP_XDEBUG: "true"
+      PHP_IDE_CONFIG: "serverName=<% if (proxy.exists) { %>app<% } else { %>www<% } %>.<%= domain %>.vm"
+      XDEBUG_CONFIG: "remote_host=${OUTRIGGER_XDEBUG_REMOTE_HOST:-localhost}"
```

Comment:
The default here should be 192.168.99.1 based on my testing. Since when this value needs to be different should be developer specific instead of project specific I would argue that we should just set that as the default in the build containers. We could either:
* Document this mechanism as the approved way to add an override when needed and just leave XDEBUG_CONFIG: to get it picked up from the developer's environment
* Set up the confd template to accept a PHP_XDEBUG_REMOTE_HOST variable it uses if that variable is not blank. In that case we'd still want to drop in the VARNAME: item here so that it can get picked up from the environment.
* Remove this line and wait for someone to figure out exactly what scenario requires setting a different value for remote_host than my testing has shown and then determine what, if anything, should be here by default. Adding this when needed, maybe in a local override for that developer only.

I'm kind of in favor of option 3 as it keeps from adding things which might not be relevant to most developers in the generated version of this file. I can file the PR for the needed first change to make it true that it is unnecessary to change the remote_host.